Lib/test/test_urllib.py

```diff
@@ -1549,7 +1549,7 @@ def test_pathname2url_win(self):
         self.assertEqual(fn('//?/unc/server/share/dir'), '//server/share/dir')
         # Round-tripping
         urls = ['///C:',
-                '///folder/test/',
```

Comment:
This should roundtrip, but it doesn't because `pathname2url('/') == '/'` (should be `'///'`). I'm planning to fix the `pathname2url()` issue and change this test case back, but probably only in 3.14.